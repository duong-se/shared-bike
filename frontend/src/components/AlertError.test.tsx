import { render, screen } from '@testing-library/react'
import { AlertError } from './AlertError'

describe('AlertError', () => {
  it('renders error correctly', async () => {
    const mockError = 'mock error string'
    render(<AlertError error={mockError} />)
    const error = (await screen.findByText(mockError)).textContent
    expect(error).toEqual(mockError)
  })

  it('does not render error and render only 1 div', async () => {
    const mockError = ''
    const { container } = render(<AlertError error={mockError} />)
    expect(container).toMatchSnapshot()
  })
})
