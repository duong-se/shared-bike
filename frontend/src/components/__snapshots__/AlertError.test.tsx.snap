// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertError does not render error and render only 1 div 1`] = `<div />`;
